build/task/test.js

```diff
@@ -16,8 +16,13 @@ galvatron.transformer
   .post('babel')
   .post('globalize');
 
+var browsers = commander.browsers || 'Firefox';
```

Comment:
If line 9's use of `[Chrome,Firefox]` in 
```text
-b, --browsers [Chrome,Firefox]', 'The browsers to run the tests in.
```
is meant to signify the defaults, then these two lines are out of sync.